include/executors/executor_sycl.hpp

```diff
@@ -257,6 +257,29 @@ class Executor<SYCL> {
    */
   Executor(cl::sycl::queue q) : q_{q} {};
 
+  cl::sycl::queue sycl_queue() const {
+    return q_;
+  }
+
```

Comment:
nip: in most cases you have 2 new lines between code blocks here you have one.

Reply:
Same class.